apollo-router/src/plugins/external.rs

```diff
@@ -43,41 +43,51 @@ struct ExternalPlugin {
     sdl: Arc<String>,
 }
 
+/// The
```

Comment:
Missing rest of comment.

Reply:
[01648ef](https://github.com/apollographql/router/pull/2173/commits/01648ef5f14e1dc394a337cdb6feaa5e74292460)